File: py_game/basic/pac/src_test/driver/basic_alone/src_test/basic/fn/set_in_range.py



"""
    Funcion que ajusta un valor en el 
    rango limite enviado.
"""
# return number
def set_inrange(value: int, range_arr: list):
    if value > range[1]: return range_arr[1]
    if value < range[0]: return range_arr[0]
    return value

File: btjs_page/to_work/old_btjs/math/mod/set_in_range.js


/*
Función que ajusta un valor en el 
rango límite enviado.
*/
export function set_in_range(VALUE, 
        RANGE_ARR) {
    if (VALUE > RANGE_ARR[1]) {
      return RANGE_ARR[1];
    }
    if (VALUE < RANGE_ARR[0]) {
      return RANGE_ARR[0];
    }
    return VALUE;
}